notebooks/monte_carlo_case_study_nigeria_data.ipynb

```diff
@@ -0,0 +1,549 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Monte-Carlo Case Study using Nigeria's Power Data\n",
+    "This Jupyter Notebook serves as a case study for analyzing simulations created using Monte-Carlo method applied to Nigeria's power data.\n",
+    "\n",
+    "## Introduction\n",
+    "\n",
+    "The goal of this notebook is to analyze the results of the Monte-Carlo simulations and gain insights into the power system of Nigeria. It utilizes the Monte Carlo method to generate multiple scenarios and assess the uncertainty in the power system's performance.\n",
+    "\n",
+    "The Monte Carlo method is a statistical technique that involves running multiple simulations with randomly sampled input parameters to estimate the distribution of the output variables. By applying this method to Nigeria's power data, we can assess the reliability and robustness of the power system under different scenarios and identify potential areas for improvement.\n",
```

Comment:
I think the method is applied rather to the model than to the data, right?

Reply:
Yes, you are right. The method is applied to the model.